code/network/multimsgs.cpp

```diff
@@ -9114,3 +9115,133 @@ void process_sexp_packet(ubyte *data, header *hinfo)
 
 	sexp_packet_received(received_packet, num_ubytes);
 }
+
+// TODO, calculate how many we can send.
+constexpr ubyte MAX_TURRET_TRACK_CHANGES = 32;
+constexpr int SIZE_LOCATION_IN_PACKET = 7;
+
+SCP_vector<int> send_turret_tracking_packet(ushort parent_sig, SCP_vector<int>& subsys_indexes) 
+{
+	Assertion(MULTIPLAYER_MASTER, "A non-server is trying to send a turret tracking packet. This is a coder mistake, please report!");
+
+	SCP_vector<int> overflow_list;
+
+	// somehow trying to send for an an invalid or non-network object
+	if (parent_sig == 0) {
+		// returning an empty list will signal the manager to clear its entries
+		return overflow_list;
+	}
+
```

Comment:
It would probably be good to verify that `subsys_indexes` isn't empty before continuing with this. Though really, it shouldn't even get here if that is the case.

Reply:
Well, that data would be nonsense because it's attached to a net signature of 0 and nothing should be done with it. So this is a way for us to gracefully tell the manager to knock it off.  But it really shouldn't try to send info if the net signature is 0 in the first place. 

How about adding an assertion that `net_signature != 0` and keeping this as a valid way to fail in the quite unlikely event that the manager tries to send this packet in release?

Reply:
Wait, you meant that I should also check if the list is empty?